bundle/deploy/terraform/init.go

```diff
@@ -218,6 +218,24 @@ func setProxyEnvVars(ctx context.Context, environ map[string]string, b *bundle.B
 	return nil
 }
 
+func setUserAgentExtraEnvVar(environ map[string]string, b *bundle.Bundle) error {
+	var products []string
+
+	if experimental := b.Config.Experimental; experimental != nil {
+		if experimental.PyDABs.Enabled {
+			products = append(products, "databricks-pydabs/0.0.0")
```

Comment:
What about the user agent in the client that the CLI uses itself?

Reply:
Do you mean "bundle run" command or any other command as well? I can fix for "bundle run" as a follow-up